src/main/java/org/prebid/server/analytics/reporter/greenbids/model/MediaTypes.java

```diff
@@ -0,0 +1,19 @@
+package org.prebid.server.analytics.reporter.greenbids.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.iab.openrtb.request.Native;
+import com.iab.openrtb.request.Video;
+import lombok.Builder;
+import lombok.Value;
+
+@Builder(toBuilder = true)
```

Comment:
Only three fields, no need for a builder. @Value(staticConstructor = "of") should be enough